domains/randomboi404.json

```
@@ -4,6 +4,8 @@
         "discord": "randomboiii5"
     },
     "records": {
-        "CNAME": "randomboi404.github.io"
+        "CNAME": "randomboi404.github.io",
+        "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
+        "TXT": ["v=spf1 include:spf.improvmx.com ~all"]
     }
```

you cant combine CNAME with other records unless proxied
```suggestion
    },
    "proxied": true
```